Octokit/Models/Response/Enterprise/PreReceiveEnvironment.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    /// <summary>
+    /// Describes a pre-receive environment.
+    /// </summary>
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class PreReceiveEnvironment
```

Comment:
response class needs parameterless ctor and ctor taking/setting all properties

Reply:
Updated